pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -61,8 +61,30 @@ const (
 
 	operationPollInterval        = 3 * time.Second
 	operationPollTimeoutDuration = 30 * time.Minute
+
+	defaultLBSourceRange = "0.0.0.0/0"
+
+	//Expected annotations for GCE
+	gceLBAllowSourceRange = "net.beta.kubernetes.io/gce-source-range"
```

Comment:
since we can include multiple comma separated ranges, please make this plural (gce-source-ranges)